assets/questions/bomberman.js

```diff
@@ -0,0 +1,150 @@
+// Copyright 2017 The TIE Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Question data for Bomberman.
+ */
+
+globalData.questions['bomberman'] = {  // eslint-disable-line dot-notation
+  title: 'Bomberman',
+  starterCode: {
+    python:
+`def bomb(board):
+    return 0
+`
+  },
+  auxiliaryCode: {
+    python:
+`class AuxiliaryCode(object):
+
+    @classmethod
+    def ignoreWalls(cls, board):
+        if not board or not board[0]:
+            return 0
+        rows = len(board)
+        cols = len(board[0])
+        maxEnemiesKilled = 0
+        enemiesOnTop = [0 for x in range(cols)]
+        enemiesOnLeft = 0
+        enemiesKilled = [[0 for x in range(cols)] for y in range(rows)]
+        for r in range(rows):
+            enemiesOnLeft = 0
+            for c in range(cols):
+                if board[r][c] == "e":
+                    enemiesOnLeft += 1
+                    enemiesOnTop[c] += 1
+                else:
+                    enemiesKilled[r][c] += enemiesOnLeft + enemiesOnTop[c]
+        enemiesOnRight = 0
+        enemiesOnBottom = [0 for x in range(cols)]
+        for r in range(rows - 1, -1, -1):
+            enemiesOnRight = 0
+            for c in range(cols - 1, -1, -1):
+                if board[r][c] == "e":
+                    enemiesOnRight += 1
+                    enemiesOnBottom[c] += 1
+                else:
+                    enemiesKilled[r][c] += enemiesOnRight + enemiesOnBottom[c]
+                    if enemiesKilled[r][c] > maxEnemiesKilled:
+                        maxEnemiesKilled = enemiesKilled[r][c]
+        return maxEnemiesKilled
+
+    @classmethod
+    def bombAllEnemiesOnBoard(cls, board):
+        if not board or not board[0]:
+            return 0
+        count = 0
+        for r in range(len(board)):
+            for c in range(len(board[0])):
+                if board[r][c] == "e":
+                    count += 1
+        return count
+`
+  },
+  tasks: [{
+    instructions: [
+      [
+        'For this question, you will be given a board, in which all',
```

Comment:
Please try to put spaces on the end of the line -- otherwise the text looks kinda wavy.